packages/engineer/src/feature/dev-server.dev-server.env.ts

```diff
@@ -227,7 +227,16 @@ devServerFeature.setup(
                 );
             }
 
-            await Promise.all(compilationPromises);
+            if (compiler.compilers.length > 0) {
```

Comment:
can we use the webpackDevMiddleware api to know when things are done?
also we need to change this entire flow to use one webpackDevMiddleware

Reply:
handled in #770